### PR TITLE
:sparkles: New Sratchpad component

### LINKED ISSUE #7: Scratchpad component
We need a scratch-board where the user can write free notes for himself.

It should consist in a title and a multi-line input.

For now, we don't want to do anything with the data. Just have it sitting there.

### PR DESCRIPTION
**Pull request by cchampou**
Original feature request is #7 

Screenshot from 2021-12-15 23-18-26

### CODE REVIEW

**Review by M123-dev — APPROVED** [review]
Thanks @cchampou, looks good

File: frontend/src/components/Card/index.tsx [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import React, { FC } from 'react';
+import './stylesheet.css';
+
+const Card: FC = ({ children }) => (
```
[the remark this answers is not in the document]
Comment (cchampou) [reply frontend/src/components/Card/index.tsx:4]: It's the functionnal component type from React. It's use to define the type of the children prop :)

**Comment by cchampou** [comment]
Awaiting merge of pull request #18

**Comment by open-longlabs** [comment]
new and simple scratchpad approved!

**Review by open-longlabs — APPROVED** [review]

**Pull request merged**